lib/widgets/emoji_reaction.dart

```diff
@@ -159,12 +159,15 @@ class ReactionChip extends StatelessWidget {
                   mainAxisSize: MainAxisSize.min,
                   crossAxisAlignment: CrossAxisAlignment.center,
                   children: [
-                    // So text-emoji chips are at least as tall as square-emoji
-                    // ones (probably a good thing).
-                    SizedBox(height: _squareEmojiScalerClamped(context).scale(_squareEmojiSize)),
                     Flexible( // [Flexible] to let text emojis expand if they can
                       child: Padding(padding: const EdgeInsets.symmetric(horizontal: 3, vertical: 1),
```

Comment:
After this change, there's 1px vertical padding on each child of the Row. Can that padding move outward?

It seems like it'd be equivalent to have instead 1px vertical padding around the Row; and then in fact it should commute with the LayoutBuilder too, and be able to merge into the outer Padding.

After that, we wouldn't need the "+ 2" adjustment, so I think we could drop the second commit and that redundant-looking `Center`.